.circleci/config.yml

```diff
@@ -348,10 +348,10 @@ jobs:
 
   # The following inherits from python-3-8. In a few cases, steps are skipped
   # based on the name of the current job (see, e.g., "Run frontend tests").
-  python-3-5:
+  python-3-6:
     <<: *job-template
     docker:
-      - image: circleci/python:3.5.9
+      - image: circleci/python:3.6.9
```

Comment:
Is it `3.6.9` or a different `3.6.x` that we should be running against?

Reply:
It didn't show since it's technically not a change, but I did mean 3.6.9. That appears to be the newest version that CircleCI provides (3.6.10 is the newest)

Reply:
But do we want the newest 3.6.x? 

I'd think we'd want the earliest 3.6.x or have some specific reason for not supporting the earliest 3.6.x

https://circleci.com/docs/2.0/docker-image-tags.json

Reply:
Ok. I was thinking that because it would be a patch release, there wouldn't be any syntax changes, so I put 3.6.9 to test against the most recent that people could be using. Guess that's a philosophical question of the intent for our CI.